### ISSUE TITLE
Why the running time in one epoch is so big

### ISSUE THREAD

**Issue by qgh1223** [Y-P-Zhang/LRF-Net-implemented-in-mmdetection#3]
Why the running time in one epoch is so big?In common mmdetection program,it is not like this.What you change the program?

**Comment by qgh1223** [comment]
Caused by the times parameter in RepeatDataset?Why changed this parameter?Is there any use？If I reduce this parameter,it will influence accuracy?

**Comment by Y-P-Zhang** [comment]
> Caused by the times parameter in RepeatDataset?Why changed this parameter?Is there any use？If I reduce this parameter,it will influence accuracy?

Yes, It may caused by "RepeatDataset".  I use this parameter because the official version of voc dataset in mmdetection do like this. It should be changed  when the dataset is coco. I will update this repository later. Thank you for reminding!

**Comment by qgh1223** [comment]
Thanks for your answering.I am customed to using the coco-like annotation style.
